substrate-archive-backend/src/frontend.rs

```diff
@@ -58,11 +80,12 @@ where
 {
 	let backend = Arc::new(ReadOnlyBackend::new(db, true));
 
-	let executor =
-		NativeExecutor::<Dispatch>::new(WasmExecutionMethod::Interpreted, Some(wasm_pages), block_workers as usize);
-
-	let executor =
-		LocalCallExecutor::new(backend.clone(), executor, Box::new(TaskExecutor::new()), Default::default())?;
+	let executor = NativeExecutor::<Dispatch>::new(
+		WasmExecutionMethod::Interpreted,
```

Comment:
why not `WasmExecutionMethod::Compiled`? It's enabled in this PR and I suppose it's faster?!

Reply:
I measured this and it's actually not faster. I was surprised by this and it might be worth taking a deeper look. When syncing the chain in normal `polkadot` using `--wasm-execution=Compiled` is much much faster.

Reply:
Ok, I see.